addons/account/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+column_copies = {
+    'account_payment_term_line': [
+        ('option', None, None),
+    ],
+}
+
+_column_renames = {
+    'account_tax': [
+        ('tax_adjustment', None),
+    ],
+    'account_tax_template': [
+        ('tax_adjustment', None),
+        ('cash_basis_account', 'cash_basis_account_id'),
+    ],
+    'account_chart_template': [
+        ('company_id', None),
+    ],
+    'account_invoice': [
+        ('reference_type', None),
+    ],
+    'account_invoice_line': [
+        ('layout_category_id', None),
+    ],
+}
+
+_table_renames = [
+    ('stock_incoterms', 'account_incoterms')
+]
+
+_model_renames = [
+    ('stock.incoterms', 'account.incoterms')
+]
+
+_field_renames = [
+    ('res.config.settings', 'res_config_settings', 'group_show_price_subtotal',
+     'group_show_line_subtotals_tax_excluded'),
+    ('res.config.settings', 'res_config_settings', 'group_show_price_total',
+     'group_show_line_subtotals_tax_included'),
+    ('res.config.settings', 'res_config_settings', 'sale_show_tax',
+     'show_line_subtotals_tax_selection'),
```

Comment:
The sale_show_tax setting is brought by the sale module in v11:
https://github.com/odoo/odoo/blob/11.0/addons/sale/models/res_config_settings.py
and is not present in the account module https://github.com/odoo/odoo/blob/11.0/addons/account/models/res_config_settings.py
Hence if you have the account module installed but not the sale module installed, I'm afraid this _field_renames will fail.

Reply:
Many methods of openupgrade, like `rename_fields`, are coded in a way that if the parameters are not found, then it does nothing. They won't fail.